Kickstarter-iOS/Views/Controllers/FindFriendsViewController.swift

```diff
@@ -33,9 +34,32 @@ internal final class FindFriendsViewController: UITableViewController {
     self.navigationController?.setNavigationBarHidden(false, animated: animated)
   }
 
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> baseTableControllerStyle()
+      |> UIViewController.lens.title %~ { _ in Strings.Follow_friends() }
+
+    _ = self.navigationController?.navigationBar
+      ?|> baseNavigationBarStyle
+
+    _ = self.loadingIndicatorView
+      |> UIActivityIndicatorView.lens.activityIndicatorViewStyle .~ .white
+      |> UIActivityIndicatorView.lens.color .~ .ksr_navy_900
+  }
+
```

Comment:
I did that because in most of the view controllers I've worked with` bindStyles()` is above `bindViewModel()`. I'll change it back though.